javascript/javascript1/week4/Voice-assistant.js

```diff
@@ -0,0 +1,113 @@
+const user = {
+  name: "",
+  todo: [],
+};
+console.log(
+  "How can I help you?",
+  "These are some Commands to warm up:",
+  "My name is Elnaz - ",
+  "What is my name - ",
+  "Add Shower to my todo - ",
+  "Remove Shower from my todo - ",
+  "What is on my todo - ",
+  "What day is it today - ",
+  "Set a timer for 1 minutes - ",
+  "How can I learn JavaScript - ",
+  "what is 3 + 3 "
+);
+function getReply(command) {
+
+  if (command.startsWith("My name is")) {
+    const name = command.replace("My name is", "").trim();
+    user.name = name;
+    console.log(`Nice to meet you ${name}`);
+  } else if (command.startsWith("What is my name")) {
+    if (user.name) {
+      console.log(user.name);
+    } else {
+      console.log(`What is your name?`);
+    }
+  } else if (command.startsWith("Add") & command.includes("to my todo")) {
+    const todo = command.replace("Add", "").replace("to my todo", "").trim();
+    user.todo.push(todo);
+    console.log(`${todo} has been added to todo`);
+  } else if (command.startsWith("Remove") & command.includes("from my todo")) {
+    const todo = command
+      .replace("Remove", "")
+      .replace("from my todo", "")
```

Comment:
Suggestion: strings are pretty easy to get wrong. Save into variables while possible.